examples/getting_started/run.py

```diff
@@ -42,23 +134,28 @@
     for i in range(NUM_STEPS):
         if config.SIMULATION_CONTROLLER.AGENT == "SphereAgent":
             obs, reward, done, info = env.step(action={
-                "add_force": np.array([5000.0, 0.0, 0.0], dtype=np.float64),
-                "add_rotation": np.array([0.0, 1.0, 0.0], dtype=np.float64)
+                "add_force": np.array([10000.0, 0.0, 0.0], dtype=np.float64),
+                "add_rotation": np.array([0.0, 0.0, 1.0])
```

Comment:
Why pyr=[0.0, 0.0, 1.0] for the rotation? Do I want a non-zero roll here? I thought I wanted a non-zero yaw.

Reply:
yaw rotation makes more sense, agreed.